lib/Bio/EnsEMBL/DataCheck/Checks/DisplayableSampleGene.pm

```diff
@@ -0,0 +1,75 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::DisplayableSampleGene;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME        => 'DisplayableSampleGene',
+  DESCRIPTION => 'Sample gene is displayable and has web_data attached to its analysis',
+  GROUPS      => ['core_handover'],
```

Comment:
There is no 'handover' group, the 'core' group is equivalent. Also makes sense to add this to the 'geneset' group.
```suggestion
  GROUPS      => ['core', 'geneset'],
```